common/src/service-connectors/matchmaking/matchmaking.types.ts

```diff
@@ -4,31 +4,40 @@ import type {ResponseStatus} from "../../global.types";
 import * as Schemas from "./schemas";
 
 export enum MatchmakingEndpoint {
-    // Scrim
-    GetQueueStatus = "GetQueueStatus",
+    GetScrim = "GetScrim",
+    GetScrimByPlayer = "GetScrimByPlayer",
+    GetScrimBySubmissionId = "GetScrimBySubmissionId",
+    GetAllScrims = "GetAllScrims",
+    GetScrimMetrics = "GetScrimMetrics",
     CreateScrim = "CreateScrim",
     JoinScrim = "JoinScrim",
     LeaveScrim = "LeaveScrim",
     CheckInToScrim = "CheckInToScrim",
-    GetAllScrims = "GetAllScrims",
-    GetScrim = "GetScrim",
-    GetScrimMetrics = "GetScrimMetrics",
-    GetScrimByPlayer = "GetScrimByPlayer",
     CompleteScrim = "CompleteScrim",
-    GetScrimBySubmissionId = "GetScrimBySubmissionId",
     CancelScrim = "CancelScrim",
     SetScrimLocked = "SetScrimLocked",
-
-    // Validation
-    ValidateReplays = "ValidateReplays",
-    ForceUpdateScrimStatus = "ForceUpdateScrimStatus",
```

Comment:
Looks like this one got removed completely; and reason for that?

Reply:
They're not used. Scrims don't hold/do submission/validation.